config/locales/fr.yml

```diff
@@ -404,6 +404,12 @@ fr:
       mascot:
         desc_html: Affiché sur plusieurs pages. Au moins 293×205px recommandé. Lorsqu’il n’est pas défini, retombe à la mascotte par défaut
         title: Image de la mascotte
+      max_bio_chars:
+        desc_html: 'Définir la taille maximum de biographies en charactères (défaut : 500)'
```

Comment:
'Longueur maximale des biographies en nombre de caractères (500 par défaut)'

Reply:
In English it was "Set the maximum biography size (default: 500)" so I went with a more direct translation. I agree with you, though; I'll update both English and French.